From 28cedc6cddd052146a5773d79a8f35ddfb4557d4 Mon Sep 17 00:00:00 2001
From: Alexandru DAMIAN <alexandru.damian@intel.com>
Date: Mon, 20 Feb 2012 13:06:48 +0200
Subject: Adds default java source version 1.5

Android uses javac target 1.5, while JDK 1.7.0 uses source
1.7 which is incompatible. Adds javac parameter source 1.5
so the compile goes as intended.

Fixes error:
javac: target release 1.5 conflicts with default source release 1.7

Issue: ANDROID-1707
Change-Id: I5c87fc8030401c05968019f15360f23b1b6e8025
---
 core/combo/javac.mk |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/combo/javac.mk b/core/combo/javac.mk
index f8c99fb..fcd98ab 100644
--- a/core/combo/javac.mk
+++ b/core/combo/javac.mk
@@ -10,9 +10,9 @@
 # Whatever compiler is on this system.
 ifeq ($(BUILD_OS), windows)
     COMMON_JAVAC := development/host/windows/prebuilt/javawrap.exe -J-Xmx256m \
-        -target 1.5 -Xmaxerrs 9999999
+        -target 1.5 -source 1.5 -Xmaxerrs 9999999
 else
-    COMMON_JAVAC := javac -J-Xmx512M -target 1.5 -Xmaxerrs 9999999
+    COMMON_JAVAC := javac -J-Xmx512M -target 1.5 -source 1.5 -Xmaxerrs 9999999
 endif
 
 # Eclipse.
-- 
1.7.4.1

